src/stories/Library/material-grid-related/material-grid-related.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { ReactComponent as TriangleIcon } from "../../../public/icons/basic/icon-triangle.svg";
+import { MaterialGrid } from "../material-grid/MaterialGrid";
+import MaterialGridData from "../material-grid/MaterialGridData";
+
+const MaterialGridRelated: React.FC = () => {
+  return (
+    <div className="material-grid-related">
+      <div className="material-grid-related__header">
+        <h2 className="material-grid-related__title">Andre materialer</h2>
+        {/* Mobile display : Dropdown */}
```

Comment:
:+1: This is helpful. 

As I read the code we are using break points to set `display: none` on the alternate version which should prevent screen readers and the like from rendering the component twice.

If you are uncertain about whether this is the case then it is probably worth double checking :smile:.